index.bs

```diff
@@ -4487,25 +4489,27 @@ partial interface MLGraphBuilder {
   <summary>
     The following pooling algorithms are supported.
   </summary>
-  <div algorithm class=algorithm-steps>
+  <div class=algorithm-steps>
+    <div algorithm>
```

Comment:
🤨 `algorithm` is a legal attribute for a `div` element? (curious, I almost never see empty attributes, with the only one I ever really see in the wild being `<input checked ...>`).

Reply:
It's a [Bikeshed thing](https://speced.github.io/bikeshed/#var-and-algorithms). It doesn't survive the processing into the output HTML. And it doesn't need to be empty - you can give it an explicit ID. It's just easier / more maintainable to let Bikeshed do the work.

Other relevant docs:
* https://garykac.github.io/procspec/#writing-algorithms
* https://dlaliberte.github.io/bikeshed-intro/#add-algorithms